REPOSITORY_GUIDE.md

```diff
@@ -232,14 +232,22 @@ source $DXOS_ROOT/dxos/tools/zsh/tools-alias.zsh
 
 ## Storybook
 
+Start a storybook dev server for any project:
+
+```bash
+pnpm nx run project:storybook
+```
+
+If you see the following error:
+
 ```
-Cannot GET /` when running 'pnpm run storybook'
+Cannot GET /` when running 'pnpm nx run project:storybook'
 ```
 
 Solution:
 
 ```bash
-pnpm run storybook --no-manager-cache
+pnpm nx run project:storybook --no-manager-cache
 ```
```

Comment:
I've been trying to get rid of this "help" for ages and it keeps coming back. I've never seen this error and I'm fairly sure it's a very old workaround. I'd just remove it.